inputfiles/overridingTypes.json

```diff
@@ -691,6 +691,9 @@
                         },
                         "nextSibling": {
                             "override-type": "ChildNode | null"
+                        },
+                        "previousSibling": {
+									"override-type": "ChildNode | null"
```

Comment:
Probably the indention should be fixed here.

Reply:
Thanks. Fixed.